### ISSUE TITLE
from builtins import bytes does not work in Python 2

### ISSUE THREAD

**Issue by moorepants** [matthias-k/cyipopt#5]
```
(opty-dev) moorepants@garuda:cyipopt-git-fork(master)$ ipython
Python 2.7.13 |Continuum Analytics, Inc.| (default, Dec 20 2016, 23:09:15) 
Type "copyright", "credits" or "license" for more information.

IPython 5.1.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import ipopt
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-151ed8d4c199> in <module>()
----> 1 import ipopt

/home/moorepants/src/cyipopt-git-fork/ipopt/__init__.py in <module>()
     39 
     40 from cyipopt import *
---> 41 from .ipopt_wrapper import minimize_ipopt
     42 

/home/moorepants/src/cyipopt-git-fork/ipopt/ipopt_wrapper.py in <module>()
      1 from __future__ import absolute_import, unicode_literals
      2 import sys
----> 3 from builtins import bytes
      4 import numpy as np
      5 try:

ImportError: No module named builtins
```

**Comment by moorepants** [comment]
This doesn't work in Python 2 or 3. I think there must be a dependency on the python package future that isn't specified anywhere.

**Comment by matthias-k** [comment]
Thanks for the report! I will look into this over the next days.

**Comment by matthias-k** [comment]
Fixed by #10. Thanks @moorepants!
